proprietary/Components/src/denhaag/link-group.tokens.json

```diff
@@ -2,15 +2,20 @@
   "denhaag": {
     "link-group": {
       "image": {
-        "padding-block-end": { "value": "{denhaag.space.block.md}" }
+        "padding-block-end": { "value": "{denhaag.space.block.md}" },
+        "size": { "value": "140px" },
+        "object-fit": { "value": "cover" }
       },
       "caption": {
         "color": { "value": "{denhaag.color.grey.5}" },
-        "font-weight": { "value": "500" }
+        "font-weight": { "value": "500" },
```

Comment:
FontWeigh 500 is still unused, I think you need to apply `{denhaag.typography.weight.bold}`

Reply:
@paulAcato font-weight **bold** is equal to **_700_**, the **_design says 500_**

Reply:
What design told me is that we simply don`t use 500.
If u check the naming (screenshot), that says 700 and overrules the font-weight given in the styles. I know this because I asked this question to Rozarin 4 months ago ( https://codefornl.slack.com/archives/C022AGFTF7Z/p1639049504019800?thread_ts=1639046592.018600&cid=C022AGFTF7Z ) 

![Screenshot 2022-03-29 at 09 14 07](https://user-images.githubusercontent.com/94894596/160554672-e7d91eea-a9f4-4ee2-865b-1c6b3f438b36.png)

Reply:
I changed it to `{denhaag.typography.weight.bold}`